docker/CMSRucioClient/scripts/syncRolesToAttributesLocalUsers.py

```diff
@@ -9,27 +9,76 @@
 from rucio.client.client import Client
 from rucio.common.exception import RSEAttributeNotFound, Duplicate, AccountNotFound, InvalidObject
 
-TO_STRIP = ['_Disk', '_Tape', '_Temp', '_Test', '_Disk_Test', '_Tape_Test', '_Ceph']
+TO_STRIP = ['_Disk', '_Tape', '_Temp', '_Test',
+            '_Disk_Test', '_Tape_Test', '_Ceph']
 
 CRIC_USERS_API = 'https://cms-cric.cern.ch/api/accounts/user/query/list/?json&preset=roles'
 CRIC_SITE_API = 'https://cms-cric.cern.ch/api/cms/site/query/?json'
+CRIC_GROUP_API = 'https://cms-cric.cern.ch/api/accounts/group/query/?json'
 PROXY = os.getenv('X509_USER_PROXY')
 
 
+def make_request(api_endpoint, cert=PROXY, verify=False):
+
+    # Pods don't like the CRIC certificate
+    result = requests.get(api_endpoint, cert=cert, verify=verify)
+    return json.loads(result.text)
+
+
 def build_site_facility_map():
     """
     Build a dictionary mapping lower case site name to lower case facility name (used as an e-mail address)
     :return:
     """
 
-    result = requests.get(CRIC_SITE_API, cert=PROXY, verify=False)  # Pods don't like the CRIC certificate
-    all_sites = json.loads(result.text)
+    # Pods don't like the CRIC certificate
+    all_sites = make_request(CRIC_SITE_API)
     site_map = {}
     for site, values in all_sites.items():
-        site_map[site.lower()] = values['facility'].lower()
+        # site-tags in in user roles use kebab-case
+        site_map[site.replace('_', '-').lower()] = values['facility'].lower()
 
     return site_map
 
+#############################################
+# RSE attibutes sync
```

Comment:
Please don't make big block comments like this. A single line in front of the function is enough. In fact, the function name is more descriptive than this comment.

Reply:
Ah, the intention was more like a visual separator for the two groups of functions.

So as in the main function, we see two major function calls that do different work. 

I can either remove them, or we can see later if moving these to separate files is a better idea.